### PR TITLE
Add button secondary disabled

### PR DESCRIPTION
**Pull request by aju**
closes https://github.com/brainly/style-guide/issues/806

screen shot 2016-07-13 at 10 39 19
screen shot 2016-07-13 at 10 39 15

### CODE REVIEW

File: src/components/buttons/_buttons_secondary.scss [hunk 1]
```diff
@@ -148,6 +151,22 @@ $includeHtml: false !default;
...
+    @include sgButtonSecondaryActive($buttonSecondaryDisabledColor, $buttonSecondaryDisabledTextColor, transparent);
+    cursor: default;
+
+    &:active {
+      color: $buttonSecondaryDisabledColor;
+      fill: $buttonSecondaryDisabledColor;
+      outline: none;
+
+      .sg-button-secondary__hole {
+        opacity: 1;
```
Comment (matzimowski) [inline src/components/buttons/_buttons_secondary.scss]: why it's needed?

Comment (matzimowski) [inline src/components/buttons/_buttons_secondary.scss]: ok, i see.

**Comment by matzimowski** [comment]
Perfect!
OK

File: src/components/buttons/_buttons_secondary.scss [hunk 2]
```diff
@@ -21,6 +21,9 @@ $buttonSecondaryActionTextColorHover: $white;
 $buttonSecondaryLightBlueColor: $blueSecondary;
 $buttonSecondaryLightBlueTextColorHover: $white;
 
+$buttonSecondaryDisabledColor: $grayLight;
```
Comment (vergilius) [inline src/components/buttons/_buttons_secondary.scss]: can I haz screenshot?

**Comment by vergilius** [comment]
👍

**Pull request merged**